project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/TopProductDataFixture.php

```diff
@@ -37,22 +37,22 @@ public function __construct(TopProductFacade $topProductFacade, Domain $domain)
      */
     public function load(ObjectManager $manager)
     {
-        $topProductReferenceNamesOnAllDomainsExcludingSecond = [
+        $defaultTopProductReferenceNames = [
             ProductDataFixture::PRODUCT_PREFIX . '1',
             ProductDataFixture::PRODUCT_PREFIX . '17',
             ProductDataFixture::PRODUCT_PREFIX . '9',
         ];
-        $topProductReferenceNamesOnSecondDomain = [
+        $distinctTopProductReferenceNames = [
             ProductDataFixture::PRODUCT_PREFIX . '14',
             ProductDataFixture::PRODUCT_PREFIX . '10',
             ProductDataFixture::PRODUCT_PREFIX . '7',
         ];
         foreach ($this->domain->getAll() as $domainConfig) {
             $domainId = $domainConfig->getId();
-            if ($domainId !== 2) {
-                $this->createTopProductsForDomain($topProductReferenceNamesOnAllDomainsExcludingSecond, $domainId);
+            if ($domainId === 2) {
```

Comment:
What about using ternary operator like this
```
$topProducts = $domainId === 2 ? $distinctTopProductReferenceNames : $defaultTopProductReferenceNames;
``` 

and then use 
```
$this->createTopProductsForDomain($topProducts, $domainId);
```

Reply:
The same occurency as were here https://github.com/shopsys/shopsys/pull/1425#discussion_r329584835

Reply:
come on, it this really so crucial? Why are doing this to me? :laughing:

Reply:
Not really crucial, but makes the code more straightforward and takes minimum time. So it make sense to me.

The condition is beceause of deciding which array to pass in `createTopProductsForDomain()`. So deciding is as much as it should do. There are no needs to call the method in both branches of conditions as you need only the right argument.

Reply:
fixed in 793ea112a